pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/plugins/ServiceBusProcessor.kt

```diff
@@ -39,8 +46,10 @@ class ServiceBusProcessor {
      */
     @Throws(BadRequestException::class)
     fun withMessage(message: ServiceBusReceivedMessage) {
+        validateJsonSchema(message)
         val sbMessageId = message.messageId
```

Comment:
The backward compatibility support for the old message field names (ex: destination_id) being transformed to the new names needs to appear before the validation step.  Otherwise the old messages will always fail validation.